doc/reference/usermode/syscalls.rst

```diff
@@ -135,6 +135,8 @@ the project out directory under ``include/generated/``:
   API call, but the sensor subsystem is not enabled, the weak handler
   will be invoked instead.
 
+* An unmarshalling function will be defined in ``include/generated/<name>_mrsh.c``
```

Comment:
Since the other bullets are expressed in the present tense (``is created``), this would read better if it were too:
```
* An unmarshalling function is defined ...
```